src/ml/data/dwh/ingestion.py

```
@@ -6,10 +6,12 @@
 """
 
 import logging
-import pandas as pd
-from pathlib import Path
-from typing import Optional, Dict, Any, List
 from datetime import datetime
+from pathlib import Path
```

Flake8 (7.2.0)

[error] 'pathlib.Path' imported but unused

(F401)